database/frontend/illustrations.json

```
@@ -1,50 +1,86 @@
+    "url": "https://www.reshot.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "StorySet",
+    "description": "Awesome free customizable illustrations for your next project.",
+    "url": "https://storyset.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "Freepik",
+    "description": "Freepik goal is to offer the best high-quality content: illustrations, photos, icons, mockups, and presentations templates etc.",
+    "url": "https://www.freepik.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "Blush Design",
+    "description": "Blush design is a creative platform offering a diverse collection of customizable illustrations for designers, developers, and content creators. With an extensive library of vibrant artwork, users can effortlessly enhance their projects. The platform allows easy customization of colors, styles, and elements to suit individual needs, empowering creativity and enhancing visual communication.",
+    "url": "https://blush.design/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "UI Store Design",
+    "description": "UI store design is a platform dedicated to providing a wide range of design resources for UI/UX professionals. It offers a curated collection of high-quality templates, icons, illustrations, and other assets to enhance the visual appeal and user experience of digital products. With a focus on simplicity and efficiency.",
+    "url": "https://www.uistore.design/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "Skribbl",
+    "description": "Enrich your creative projects and support free access to art of education with the collection of hand-drawn illustrations.",
+    "url": "https://weareskribbl.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "Humaaans",
+    "description": "Humaaans is a flexible illustration library that simplifies visual storytelling. With diverse customizable human figures, it offers a wide range of poses and clothing options for inclusive and unique illustrations. Create captivating visuals effortlessly for websites, presentations, and more. Customize, mix, and match vector illustrations to bring your stories to life.",
+    "url": "https://humaaans.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "Open Doodles",
+    "description": "Open Doodles is a vibrant collection of freely-usable illustrations that add personality to any project. With diverse hand-drawn characters, objects, and backgrounds, Open Doodles sparks creativity for designers, developers, and content creators. Enhance websites, presentations, and social media with these charming doodles. Let your imagination flow with the expressive world of Open Doodles.",
```

[praise] I like how you mention that these illustrations enhance projects! :) Well done! 👍🏾 
```suggestion
    "description": "Open Doodles is a vibrant collection of freely-usable illustrations that add personality to any project. With diverse hand-drawn characters, objects, and backgrounds, Open Doodles sparks creativity for designers, developers, and content creators. Enhance websites, presentations, and social media with these charming doodles. Let your imagination flow with the expressive world of Open Doodles.",
```